src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypingGivenAllelesUtils.java

```diff
@@ -35,10 +36,9 @@ public static VariantContext composeGivenAllelesVariantContextFromRod(final Feat
                                                                           final boolean snpsOnly,
                                                                           final Logger logger,
                                                                           final FeatureInput<VariantContext> allelesBinding) {
-        if ( tracker == null ) throw new IllegalArgumentException("the tracker cannot be null");
-        if ( loc == null ) throw new IllegalArgumentException("the location cannot be null");
-        if ( allelesBinding == null ) throw new IllegalArgumentException("the alleles binding cannot be null");
-
+        Utils.nonNull(tracker);
```

Comment:
add message to all null checks (only if there are more then 1 in a method, as here)